airflow-core/src/airflow/cli/commands/db_manager_command.py

```diff
@@ -18,33 +18,50 @@
 
 from airflow import settings
 from airflow.cli.commands.db_command import run_db_downgrade_command, run_db_migrate_command
-from airflow.providers.fab.auth_manager.models.db import _REVISION_HEADS_MAP, FABDBManager
+from airflow.configuration import conf
 from airflow.utils import cli as cli_utils
+from airflow.utils.module_loading import import_string
 from airflow.utils.providers_configuration_loader import providers_configuration_loaded
 
 
+def _get_db_manager(name):
+    """Import the db manager class."""
+    db_managers = conf.get("database", "external_db_managers")
+    db_manager = None
+    for manager in db_managers.split(","):
+        if name in manager:
+            db_manager = manager
+            break
+    if db_manager is None:
+        raise SystemExit(f"DB manager {name} not found in configuration.")
+    return import_string(db_manager.strip())
```

Comment:
```suggestion
def _get_db_manager(manager_classpath: str):
    """Import the db manager class."""
    db_managers = conf.getlistlist("database", "external_db_managers")
    if manager_classpath not in manager:
        raise SystemExit(f"DB manager {manager_classpath} not found in configuration.")
    return import_string(manager_classpath.strip())
```

Something like this is probably better.

Reply:
It might not be nice to have a command like:
`airflow db-manager "airflow.providers.fab.auth_manager.models.db.FABDBManager" reset -y`
or what do you think?